Packages/vcs/Lib/vcs2vtk.py

```diff
@@ -1288,11 +1286,10 @@ def prepGlyph(g, marker, index=0):
 
 def setMarkerColor(p, marker, c, cmap=None):
     # Color
-    if cmap is None:
-        if marker.colormap is not None:
-            cmap = marker.colormap
-        else:
-            cmap = 'default'
+    if marker.colormap is not None:
```

Comment:
if cmap is not None then should you do this instead:

# Check if cmap is None
if cmap is None and maker.colormap is not None:
  cmap = ***
elif cmake is None and vcs._colorMap is not None:
  cmap = ***
else:
  cmap = 'default' # Do we need need default?